sentinel-cluster/sentinel-cluster-server-default/src/main/java/com/alibaba/csp/sentinel/cluster/server/NettyTransportServer.java

```diff
@@ -50,15 +49,17 @@
  */
 public class NettyTransportServer implements ClusterTokenServer {
 
+    private static final int DEFAULT_BOSS_EVENT_LOOP_THREADS = 1;
     private static final int DEFAULT_EVENT_LOOP_THREADS = Math.max(1,
         SystemPropertyUtil.getInt("io.netty.eventLoopThreads", Runtime.getRuntime().availableProcessors() * 2));
+    private static final boolean useEpoll = SystemPropertyUtil.getBoolean("io.netty.epoll", false);
```

Comment:
I suggest moving it to `start()` and users maybe easier to set the properties dynamically.
How do you think it?

Reply:
get property in the <clinit> because DEFAULT_EVENT_LOOP_THREADS get property also in the <clinit>
but you are right,i will move it to start() method